aiida/orm/data/array/trajectory.py

```diff
@@ -135,11 +146,25 @@ def set_trajectory(self, stepids, cells, symbols, positions, times=None, velocit
 
         .. todo :: Choose suitable units for velocities
         """
+
+        import numpy
+
         self._internal_validate(stepids, cells, symbols, positions, times, velocities)
-        self.set_array('steps', stepids)
-        self.set_array('cells', cells)
-        self.set_array('symbols', symbols)
+        # set symbols as attribute for easier querying
```

Comment:
Here we can do `list(symbols)` to make sure it's a list?

Reply:
Yep, implemented that in 016a59d05265924edf68b46db30b79d80a0cae82